tests/test_validators.py

```diff
@@ -0,0 +1,95 @@
+import pytest
+
+from log_outgoing_requests.validators import (
+    _validate_content_length,
+    _validate_content_type,
+)
+
+
+#
+# Fixtures
+#
+@pytest.fixture()
+def config():
+    from log_outgoing_requests.models import OutgoingRequestsLogConfig
+
+    conf = OutgoingRequestsLogConfig.get_solo()
+    return conf
+
+
+@pytest.fixture()
+def config_max_content_length(config):
+    config.max_content_length = 150
+    config.save()
+    return config
+
+
+@pytest.fixture()
+def config_content_types(config):
+    config.allowed_content_types = ["text/*", "application/json"]
+    config.save()
+    return config
+
+
+#
+# Test _validate_content_length
+#
+@pytest.mark.django_db
+class TestValidateContentLength:
```

Comment:
organizing by class is not really idiomatic pytest, preferably you just define test functions and split them into separate files/python modules. The `self` attribute sits there cluttering the method signature and it's unused. Pytest functions are more elegant and less bloated, and encourage test isolation due to not having a shared base available.